OpenDreamShared/Network/Messages/MsgWinClone.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Lidgren.Network;
+using Robust.Shared.Network;
+using Robust.Shared.Serialization;
+
+namespace OpenDreamShared.Network.Messages;
+
+public sealed class MsgWinClone : NetMessage {
```

Comment:
This needs a `_netManager.RegisterNetMessage<MsgWinClone>();` in `DreamManager.Connections.cs` before it can be sent to clients.